state/protocol/inmem/encodable.go

```diff
@@ -51,3 +52,10 @@ type EncodableCluster struct {
 	RootBlock *cluster.Block
 	RootQC    *flow.QuorumCertificate
 }
+
+// EncodableParams is the encoding format for protocol.GlobalParams
```

Comment:
Not sure the `godoc` correctly reflects the `EncodableParams` purpose.

Reply:
`EncodableParams` is just a concrete (`struct` type) representation of `protocol.GlobalParams` (`interface` type), mainly for encoding.